crates/op-consensus/src/receipt/receipts.rs

```diff
@@ -0,0 +1,271 @@
+use super::OpTxReceipt;
+use alloy_consensus::TxReceipt;
+use alloy_primitives::{Bloom, Log};
+use alloy_rlp::{length_of_length, BufMut, Decodable, Encodable};
+
+#[cfg(not(feature = "std"))]
+use alloc::vec::Vec;
+
+/// Receipt containing result of transaction execution.
+#[derive(Clone, Debug, PartialEq, Eq, Default)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
+pub struct OpReceipt<T = Log> {
```

Comment:
this should be `OpDepositReceipt`
which wraps the alloy-consensus receipt and adds extra fields for deposit